pages/Auto-Disenchanter.md

```diff
@@ -0,0 +1,3 @@
+# Auto Disenchanter
+It requires a source of electricity, see [Electric Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
+You place the slimefun or vanilla tool in the left slot along with a book, and an indicator will appear showing the progress on the process. Upon completion, the right side will produce the respective tool, disenchanted, and an enchanted book holding the enchants of that tool.
```

Comment:
There should be a `## Usage` heading before this sentence.

Reply:
Edit performed, synced the repo so that I don't roll anything back by accident.